packages/ui/src/validators/hooks/useValidatorMembers.tsx

```diff
@@ -0,0 +1,66 @@
+import { useMemo } from 'react'
+import { map } from 'rxjs'
+
+import { useApi } from '@/api/hooks/useApi'
+import { useFirstObservableValue } from '@/common/hooks/useFirstObservableValue'
+import { useGetMembersWithDetailsQuery } from '@/memberships/queries'
+import { asMemberWithDetails } from '@/memberships/types'
+
+import { ValidatorMembership } from '../types'
+
+export const useValidatorMembers = () => {
+  const { api } = useApi()
+  const allValidatorAddresses = useFirstObservableValue(
+    () =>
+      api?.query.staking.validators
+        .entries()
+        .pipe(map((entries) => entries.map((entry) => entry[0].args[0].toString()))),
+    [api?.isConnected]
+  )
+
+  const allValidatorsWithCtrlAcc = useFirstObservableValue(
+    () =>
+      allValidatorAddresses &&
+      api &&
+      api.query.staking.bonded
+        .multi(allValidatorAddresses)
+        .pipe(map((entries) => entries.map((entry) => (entry.isSome ? entry.unwrap().toString() : undefined)))),
+    [allValidatorAddresses, api?.isConnected]
+  )
+
+  const variables = {
+    where: {
+      boundAccounts_containsAny:
+        (allValidatorsWithCtrlAcc?.concat(allValidatorAddresses).filter((element) => !element) as string[]) ?? [],
```

Comment:
Please address: https://github.com/Joystream/pioneer/pull/4539#discussion_r1404561384

Also did you check https://github.com/Joystream/pioneer/pull/4539#discussion_r1404563471 ?

Reply:
> Please address: https://github.com/Joystream/pioneer/pull/4539#discussion_r1404561384

I addressed it.

> Also did you check https://github.com/Joystream/pioneer/pull/4539#discussion_r1404563471 ?

Yes, I checked it. We can't get the controller account using this api.
![queryResult](https://github.com/Joystream/pioneer/assets/129978066/1ef4dd44-4c5e-4047-87af-0dfee79ee09f)